packages/cli/cli/src/persistence/storeOutputDirectories.ts

```diff
@@ -0,0 +1,8 @@
+import { mkdir, writeFile } from "fs/promises";
+import path from "path";
+import { getPathToOutputDirectoriesFile } from "./getPathToOutputDirectoriesFile";
+
+export async function storeOutputDirectories(outputDirectories: string[]): Promise<void> {
```

Comment:
How do you handle relative/absolute filepaths here? i.e. if a directory has been approved and its a relative path then should we normalize to absolute here?

Reply:
we have some type safe wrappers called `AbsoluteFilepath` and `RelativeFilepath` that may be helpful and also account for windows/mac/etc

Reply:
We normalize to absolute when we check the filepath, and only pass in absolute filepaths here, I'll make that clear by updating the type of outputDirectories to AbsoluteFilepath